web-registry/src/components/templates/ProjectDetails.tsx

```diff
@@ -125,34 +126,35 @@ function ProjectDetails(): JSX.Element {
 
   useEffect(() => {
     setPageView(location);
+    setGeojson(null); // reset when location changes
   }, [location]);
 
   const otherProjects = projectsData?.allProjects?.nodes?.filter(
     p => p?.handle !== projectId,
   );
 
-  const [geojson, setGeojson] = useState<any | null>(null);
-
   // Convert kml to geojson
   const mapFile: string =
     metadata?.['http://regen.network/boundaries']?.['@value'];
   const isGISFile: boolean = /\.(json|kml)$/i.test(mapFile);
   const isKMLFile: boolean = /\.kml$/i.test(mapFile);
 
-  if (!geojson && isGISFile) {
-    fetch(mapFile)
-      .then(r => r.text())
-      .then(text => {
-        let geojson;
-        if (isKMLFile) {
-          const dom = new DOMParser().parseFromString(text, 'text/xml');
-          geojson = togeojson.kml(dom);
-        } else {
-          geojson = JSON.parse(text);
-        }
-        setGeojson(geojson);
-      });
-  }
+  useEffect(() => {
+    if (!geojson && isGISFile) {
+      fetch(mapFile)
```

Comment:
if there's no map file, we should use the `regen:location` geojson value instead

Reply:
done

Reply:
could you remove the http://regen.network/boundaries field from the "123" project on staging so we can make sure it works as expected?